lib/streamlit/runtime/caching/memo_decorator.py

```diff
@@ -217,7 +217,7 @@ def __call__(self, func: F) -> F:
     def __call__(
         self,
         *,
-        persist: str | None = None,
+        persist: str | bool | None = None,
```

Comment:
I think this is a very minor thing, but since we expect only a "disk" string here, instead of `str` we could be more precise and say `Literal["disk"]` here.

Reply:
Yes, I like this!